docs/en/middleware.rst

```diff
@@ -201,25 +201,88 @@ Both redirect handlers share the same configuration options:
 
 For example::
 
+    use Authorization\Exception\MissingIdentityException;
+    
     $middlewareQueue->add(new AuthorizationMiddleware($this, [
         'unauthorizedHandler' => [
             'className' => 'Authorization.Redirect',
-            'url' => '/users/login',
+            'url' => '/pages/unauthorized',
             'queryParam' => 'redirectUrl',
             'exceptions' => [
                 MissingIdentityException::class,
                 OtherException::class,
             ],
         ],
     ]));
-
-You can also add your own handler. Handlers should implement
-``Authorization\Middleware\UnauthorizedHandler\HandlerInterface``, be suffixed
-with ``Handler`` suffix and reside under your app's or plugin's
-``Middleware\UnauthorizedHandler`` namespace.
+    
+Handlers catch ONLY those exceptions which extend the 
+``Authorization\Exception\Exception`` class.
+If you want to catch any other exceptions which should
+be handled via your handler they need to be added to
+the ``execeptions`` array
```

Comment:
That reads a little misleading, as if adding exceptions to the `exceptions` config would change the behavior that only exceptions that do extend `\Authorization\Exception\Exception` are being caught, which isn't the case.

Reply:
OK, yes sorry.
I meant the default RedirectHandler (and anything extending it) only catch exceptions from that class (or exceptions based on that)
https://github.com/cakephp/authorization/blob/master/src/Middleware/UnauthorizedHandler/RedirectHandler.php#L61
I will reword that.

Reply:
I probably should have explained my reasoning a little more. The type restriction applies to all handlers, `HandlerInterface::handle()` only accepts that type of exception, also the middleware only catches exceptions of that type in the first place. So no matter the handler config, they can only ever receive exceptions that extend `\Authorization\Exception\Exception`.

Reply:
Ohhh, so its not a restriction of that specific handler, its a restriction of the plugin. I see

Reply:
I hope my explaination doesn't confuse people more 😅